tests/ui/rfcs/rfc-2632-const-trait-impl/effects/trait-fn-const.rs

```diff
@@ -0,0 +1,15 @@
+// Regression test for issue #113378.
+// FIXME(effects): This test can probably be removed once the `effects` lowering
+// is enabled by default since we already have a bunch of tests for this elsewhere.
```

Comment:
```suggestion
```

Right I wrote this before improving the diagnostic for code using `const_trait_impl`. This paragraph is now outdated.